ReactNativeClient/lib/services/KeymapService.ts

```diff
@@ -0,0 +1,317 @@
+		{ accelerator: 'Cmd+A', command: 'selectAll' },
+		{ accelerator: 'Cmd+B', command: 'textBold' },
+		{ accelerator: 'Cmd+I', command: 'textItalic' },
+		{ accelerator: 'Cmd+K', command: 'textLink' },
+		{ accelerator: 'Cmd+`', command: 'textCode' },
+		{ accelerator: 'Shift+Cmd+T', command: 'insertDateTime' },
+		{ accelerator: 'Shift+Cmd+F', command: 'focusSearch' },
+		{ accelerator: 'Cmd+F', command: 'showLocalSearch' },
+		{ accelerator: 'Shift+Cmd+S', command: 'focusSidebar' },
+		{ accelerator: 'Shift+Cmd+L', command: 'focusNoteList' },
+		{ accelerator: 'Shift+Cmd+N', command: 'focusNoteTitle' },
+		{ accelerator: 'Shift+Cmd+B', command: 'focusNoteBody' },
+		{ accelerator: 'Option+Cmd+S', command: 'toggleSidebar' },
+		{ accelerator: 'Cmd+L', command: 'toggleVisiblePanes' },
+		{ accelerator: 'Cmd+0', command: 'zoomActualSize' },
+		{ accelerator: 'Cmd+E', command: 'startExternalEditing' },
+		{ accelerator: 'Option+Cmd+T', command: 'setTags' },
+		{ accelerator: 'Cmd+G', command: 'gotoAnything' },
+		{ accelerator: 'F1', command: 'help' },
+	];
+
+	static readonly defaultKeymap_OtherPlatforms: KeymapItem[] = [
+		{ accelerator: 'Ctrl+N', command: 'newNoteItem' },
+		{ accelerator: 'Ctrl+T', command: 'newTodoItem' },
+		{ accelerator: 'Ctrl+S', command: 'synchronize' },
+		{ accelerator: 'Ctrl+P', command: 'printItem' },
+		{ accelerator: 'Ctrl+Q', command: 'quit' },
+		{ accelerator: 'Ctrl+Alt+I', command: 'insertTemplate' },
+		{ accelerator: 'Ctrl+C', command: 'copy' },
+		{ accelerator: 'Ctrl+X', command: 'cut' },
+		{ accelerator: 'Ctrl+V', command: 'paste' },
+		{ accelerator: 'Ctrl+A', command: 'selectAll' },
+		{ accelerator: 'Ctrl+B', command: 'textBold' },
+		{ accelerator: 'Ctrl+I', command: 'textItalic' },
+		{ accelerator: 'Ctrl+K', command: 'textLink' },
+		{ accelerator: 'Ctrl+`', command: 'textCode' },
+		{ accelerator: 'Ctrl+Shift+T', command: 'insertDateTime' },
+		{ accelerator: 'F6', command: 'focusSearch' },
+		{ accelerator: 'Ctrl+F', command: 'showLocalSearch' },
+		{ accelerator: 'Ctrl+Shift+S', command: 'focusSidebar' },
+		{ accelerator: 'Ctrl+Shift+L', command: 'focusNoteList' },
+		{ accelerator: 'Ctrl+Shift+N', command: 'focusNoteTitle' },
+		{ accelerator: 'Ctrl+Shift+B', command: 'focusNoteBody' },
+		{ accelerator: 'F10', command: 'toggleSidebar' },
+		{ accelerator: 'Ctrl+L', command: 'toggleVisiblePanes' },
+		{ accelerator: 'Ctrl+0', command: 'zoomActualSize' },
+		{ accelerator: 'Ctrl+E', command: 'startExternalEditing' },
+		{ accelerator: 'Ctrl+Alt+T', command: 'setTags' },
+		{ accelerator: 'Ctrl+,', command: 'config' },
+		{ accelerator: 'Ctrl+G', command: 'gotoAnything' },
+		{ accelerator: 'F1', command: 'help' },
+	];
+
+	static readonly defaultKeymap = shim.isMac()
+		? KeymapService.defaultKeymap_macOS
+		: KeymapService.defaultKeymap_OtherPlatforms;
+
+	constructor() {
+		super();
+
+		this.keymap = KeymapService.initKeymap();
+		this.logger_ = new Logger();
```

Comment:
If the class overrides BaseService, that should not be needed. You can access `this.logger()` from BaseService.

Reply:
Sorry about that. I'll remove it.